src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -210,39 +227,59 @@ public DashboardWebApplication(Action<WebApplicationBuilder>? configureBuilder =
         _app.MapGrpcService<OtlpLogsService>();
     }
 
+    /// <summary>
+    /// Load <see cref="DashboardOptions"/> from configuration without using DI. This performs
+    /// the same steps as getting the options from DI but without the need for a service provider.
+    /// </summary>
+    private static DashboardOptions GetDashboardOptions(WebApplicationBuilder builder, IConfigurationSection dashboardConfigSection)
+    {
+        var dashboardOptions = new DashboardOptions();
+        dashboardConfigSection.Bind(dashboardOptions);
+        new PostConfigureDashboardOptions(builder.Configuration).PostConfigure(name: string.Empty, dashboardOptions);
+        var result = new ValidateDashboardOptions().Validate(name: string.Empty, dashboardOptions);
+        if (result.Failed)
+        {
+            throw new OptionsValidationException(optionsName: string.Empty, typeof(DashboardOptions), result.Failures);
+        }
```

Comment:
Does this look reasonable to the user who launches this standalone?

Reply:
It's a little funky. The exception message is all the failure messages joined together with semi-colons. It's informative but ugly. Probably also a big stack trace.

When P5 docker image is published I'll see what it looks like and what can be done to improve it.

Reply:
Issue: https://github.com/dotnet/aspire/issues/3168